.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v1
```

Comment:
Let's also make sure to bump all these to `actions/setup-go@v4` to fix deprecation warnings.